src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +51,161 @@ export const KafkaPage: FunctionComponent = () => {
-                {t("kafkaoverview:purchaseCardCallToActionButton")}
+                {t("purchaseCardCallToActionButton")}
                 <ExternalLinkAltIcon className="pf-u-ml-md" />
               </Button>
             </CardFooter>
           </Card>
           <Card>
             <CardHeader>
               <CardTitle>
-                <Title headingLevel="h3">
-                  {t("kafkaoverview:contactSalesCardTitle")}
-                </Title>
+                <Title headingLevel="h3">{t("contactSalesCardTitle")}</Title>
               </CardTitle>
             </CardHeader>
-            <CardBody>{t("kafkaoverview:contactSalesCardMainText")}</CardBody>
+            <CardBody>{t("contactSalesCardMainText")}</CardBody>
             <CardFooter>
               <Button
                 data-testid="cardContactSales-buttonCTA"
-                variant={ButtonVariant.secondary}
+                variant={ButtonVariant.link}
+                isLarge
+                isInline
                 component="a"
-                href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-streams-for-apache-kafka#contact-sales"
+                href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-streams-for-apache-kafka#contact-us"
                 target="_blank"
               >
-                {t("kafkaoverview:contactSalesCardCallToActionButton")}
+                {t("contactSalesCardCallToActionButton")}
                 <ExternalLinkAltIcon className="pf-u-ml-md" />
               </Button>
             </CardFooter>
           </Card>
         </Grid>
       </MarketingPageSection>
+      <MarketingPageSection>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h3">{t("pricingModalTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Flex>
+              <Flex flex={{ default: "flex_1" }}>
+                <FlexItem>
+                  <DescriptionList
+                    isHorizontal
+                    className="description-list-spacer"
+                  >
+                    <DescriptionListGroup className="description-list-spacer">
+                      <DescriptionListTerm>
+                        {t("steamingUnit")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$1.49/hour"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup className="description-list-spacer">
+                      <DescriptionListTerm>
+                        {t("dataTransfer")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.09/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup className="description-list-spacer">
```

Comment:
When you make the suggested updates above to use `alignSelf` and `justifyContent`, you can then remove the class `description-list-spacer` from all components above, since it should no longer be needed to adjust the positioning of these contents.